org.jellyfin.JellyfinServer.Plugin.IntelComputeRuntime.yml

```diff
@@ -0,0 +1,134 @@
+app-id: org.jellyfin.JellyfinServer.Plugin.IntelComputeRuntime
+branch: "1"
```

Comment:
What's the reason for using it for the branch name? As it targets single app, `stable` makes more sense to me.

Reply:
There is no need for it to be 1. I changed it to stable.